src/basic/string-util.c

```diff
@@ -583,6 +583,24 @@ static void advance_offsets(
                 shift[1] += size;
 }
 
+char *strip_newline(char **ibuf, size_t *_isz) {
+        assert(ibuf);
+        assert(*ibuf);
+
+        char *p = *ibuf;
+        size_t len = 0;
+
+        while (*p & ~'\n') {
+                len++;
+                p++;
+        }
+
+        *p = '\0';
+        *_isz = len;
```

Comment:
you can exclude "len" variable at all and calculate "*_isz" so:
*_isz = p - ibuf;

Reply:
Thank you. Fixed in 32325564557be491e7b20800d19d66df53bb970e